force-app/main/default/aura/RD2_EntryForm/RD2_EntryFormHelper.js

```diff
@@ -1,17 +1,4 @@
 ({
-    /**
-     * @description: Create the modal when initializing the component
-     */
-    handleShowModal: function (component, event, helper) {
-        let recordId = component.get('v.recordId');
-        let parentId;
-
-        if (!recordId) {
-            parentId = helper.getParentId();
-            component.set('v.parentId', parentId);
```

Comment:
I think we need to keep this assignment statement (and the inContextOfRef code), as it is responsible for setting the Contact/Account lookup in the RD2 Entry Form Modal when opening it from a Related List.

Reply:
reverted the commit